client/packages/core/src/Reactor.js

```diff
@@ -1217,7 +1231,14 @@ export default class Reactor {
   // Rooms
 
   joinRoom(roomId) {
-    this._trySendAuthed(uuid(), { op: "join-room", "room-id": roomId });
+    this._rooms[roomId] = {
+      isLoading: true,
```

Comment:
Same as above, I wonder how this will function if there are two places in one page that call `joinRoom`. 

Perhaps we should de-dupe?